Update datadog instructions (#277)
RoadieHQ/marketing-site: content/backstage/plugins/datadog.md

        </EntityPageLayout>
       )

---

## Specify datadog domain

> Datadog embedded graph is using datadoghq.euas default top-level domain, when other is not specified. If you are using other domain, you need to specify it with corresponding annotations datadoghq.com/site.

Reviewer: ```suggestion
Datadog embedded graph is using datadoghq.eu as the default top-level domain. If you are using another domain, you must specify it with corresponding annotations datadoghq.com/site.
```
